esp/services/ws_config2/ws_config2Session.hpp

```diff
@@ -80,7 +80,7 @@ struct ConfigMgrSession {
 
     void getEnvironmentFullyQualifiedPath(const std::string &envFile, std::string &fullPath)
     {
-        fullPath = sourcePath + PATHSEPSTR + envFile;
+        fullPath = sourcePath + envFile;
```

Comment:
I'm not sure where envFile is coming from, but if it's coming from client input, let's make sure we're handling the presence and/or absence of the path seperator

Reply:
The sourcePath, and other paths are checked when set (see lines around 57 in ws_config2Service.cpp), to ensure that they end with a separator. The input should just be a filename as returned from the get environment file list.